test/common.bash

```diff
@@ -31,6 +31,7 @@ GC_SOURCES="${GC_LOC}/*.v3"
 V3C_HEAP_SIZE=${V3C_HEAP_SIZE:="-heap-size=500m"}
 
 PROGRESS=${VIRGIL_LOC}/test/config/progress
```

Comment:
What about just adding the `PROGRESS_ARGS` here so that it doesn't need to be threaded through all the individual test files? (just removed)

Reply:
Hm? Not sure I follow. Do you mean change the `PROGRESS`  env variable to effectively `progress $PROGESS_ARGS`?

Reply:
Done.